images/hail-gcp-notebook/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
```

Do you have a tag to pin against, even if it's explicitly `:latest`?

```suggestion
FROM gcr.io/deeplearning-platform-release/base-cpu:latest
```

---

Yes, good point, thanks! I've pinned the version to `m93`, which is the most recent one.